Mod0/syllabus.md

```diff
@@ -77,23 +80,16 @@ There will be no “gotchas”. The rubric that follows communicates what we car
-- **USB-C to HDMI Cable:** This cable will allow you to connect your MacBook to your external monitor/display. If you are purchasing a new monitor, it will likely come with the cable to you, but you may need to aquire one.
+- **External Monitor/Display:** You will receive an external monitor along with your laptop. You will appreciate the additional screen space using all the tools as you navigate your new career. An HDMI-to-HDMI cable (i.e. full-size HDMI plug on both ends) is included in the monitor box to connect your laptop to the monitor.
 - **Dedicated Workspace:** It is crucial to have a workspace that has access to a reliable internet connection and is both quiet and free from distractions. Take time before Mod 0 to build your workspace and experiment with different setups until you find a configuration that works for you.
-- **Notebook + Writing Utensils:** While you will be working on your laptop while you code, work on projects, etc., we recommend using a physical notebook to take notes during class and workshops.
+- **Notebook + Writing Utensils:** While you will be working on your laptop while you code, work on projects, etc., we recommend using a physical notebook to take notes during class and workshops. There are apps available 
 
 ## Optional Supplies
 - **Headphones with Microphone:** While your Turing-issued laptop has built-in speakers and a microphone, if your workspace has a lot of ambient noise, it might be a good idea to invest in headphones + microphone. 
-- **USB-C Hub:** Your Turing-issued laptop only has 2 ports. You'll need one for your charger and one for your external monitor. If you require more ports, a USB-C Hub is a great idea!
+- **USB-C Hub:** Your Turing-issued laptop has a limited number of ports. One is for your charged and another is for your external monitor. If you require more ports, a USB-C Hub is a great idea!
 - **Laptop Stand:** These can be useful when using a separate keyboard and for preventing strain on your neck. This also creates more real estate for your workspace.
 - **External Keyboard:** You may benefit from using a different keyboard than the integrated one in your laptop, especially if you are using a laptop stand. If you plan to use a wired keyboard, you'll likely need the USB-C hub mentioned above to connect to your keyboard in addition to your charger and monitor.
 
-
-<!-- ## Resources
-- Turing Lingo
-- JavaScript Cheat Sheet
-- Ruby Cheat Sheet -->
-
 ## Accommodations
 We do our best to work with students who need additional accommodations while attending Turing. If this applies to you, please review [this form](https://frontend.turing.edu/documents/reasonable_accommodation_request_form.pdf) prior to starting Mod 0 at Turing and communicate with the Mod 0 Lead. 
```

Comment:
This is very small thing, but I would change from Mod 0 lead to one of your instructors.

Reply:
Good call out. The original intro paragraph identified another Turing employee as the Mod 0 Lead, and all roads led to her.